ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,461 @@
+---
```

Comment:
```suggestion
---
eip: 8109
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
Thank you @abcoathup